app/db/migrations/versions/c4b9d0057513_case_adaptations.py

```diff
@@ -0,0 +1,73 @@
+"""case adaptations
+
+Revision ID: c4b9d0057513
+Revises: 117a6ec2770b
+Create Date: 2024-11-11 14:37:55.834314
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from alembic_postgresql_enum import TableReference
+
+# revision identifiers, used by Alembic.
+revision: str = "c4b9d0057513"
+down_revision: Union[str, None] = "117a6ec2770b"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "case_adaptations",
+        sa.Column("created_at", sa.DateTime(), nullable=False),
+        sa.Column("updated_at", sa.DateTime(), nullable=False),
+        sa.Column("id", sa.Uuid(), nullable=False),
+        sa.Column("needed_adaptations", sa.JSON(), nullable=True),
+        sa.Column("languages", sa.JSON(), nullable=True),
+        sa.Column("case_id", sa.Uuid(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["case_id"],
+            ["cases.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
```

Comment:
The order these run in dictates the order of the columns in the table.

You may wish to manually re-order these if you wish.